### PR TITLE
added 2 speakers

### PR DESCRIPTION
**Pull request by prankshaw**
Added speaker Sudhir Verma and Damini Saty

### CODE REVIEW

**Comment by prankshaw** [comment]
Resolved Issue **Add speakers Sudhir Verma & Damini Satya #418**

**Comment by prankshaw** [comment]
If any changes are required , Please tell me

File: index.html [hunk 1]
```diff
@@ -3665,6 +3665,42 @@ <h1>Speakers</h1>
...
+                                   <div class="hover-state text-center preserve3d">
+                                       <div class="social-links vertical-align">
+                                           <a target="default" href="https://github.com/redhat-developer-tooling/developer-platform-install"><i class="icon fa fa-external-link"></i></a>
+                                           <a target="default" href="https://github.com/sudhirverma"><i class="icon fa fa-github"></i></a>
+                                       </div>
+                                   </div>
+                               </div>
+                               <span class="speaker-name">Sudhir Verma</span>
+                               <span>Red Hat</span>
+                               <span></span>
```
Comment (simsausaurabh) [inline index.html]: Please ask the company of Sudhir Verma to @edenyay @hpdang

File: index.html [hunk 2]
```diff
@@ -3665,6 +3665,42 @@ <h1>Speakers</h1>
                             </div>
                         </div>
 
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                           <div class="speaker">
+                               <div class="image-holder">
+                                   <img class="background-image" alt="Sudhir Verma" src="img/Sudhir Verma.jpg">
```
Comment (simsausaurabh) [inline index.html]: Remove space from image name

File: index.html [hunk 3]
```diff
@@ -3665,6 +3665,42 @@ <h1>Speakers</h1>
...
+                               <span class="speaker-name">Sudhir Verma</span>
+                               <span>Red Hat</span>
+                               <span></span>
+                           </div>
+                       </div>
+
+                       <div class="col-md-3 col-sm-6 speaker-column">
+                          <div class="speaker">
+                              <div class="image-holder">
+                                  <img class="background-image" alt="Damini Satya" src="img/Damini Satya.jpg">
```
Comment (simsausaurabh) [inline index.html]: Remove space from image name

File: index.html (hunk 1 above)
Comment (prankshaw) [reply index.html]: should I ask It here or somewhere else

**Comment by prankshaw** [comment]
@edenyay @hpdang please tell me the company of speaker Sudhir Verma , as only Red Hat is mentioned and I need another detail about him as it has been asked to me

**Comment by prankshaw** [comment]
solved issue
**Speaker Images Not aligned properly #420**
**Add speakers Sudhir Verma & Damini Satya**

**Comment by prankshaw** [comment]
preview -
https://prankshaw.github.io/2018-fossasia/

**Comment by prankshaw** [comment]
**Add speakers Sudhir Verma & Damini Satya #418** resolved
please see it in preview link mentioned above

**Comment by prankshaw** [comment]
@nityanandagohain I tried chnging size of Chetan Khatri image to 300*300px but it was not working properly then

**Comment by prankshaw** [comment]
please review my changes now

File: index.html (hunk 1 above)
Comment (hpdang) [reply index.html]: @prankshaw Software Engineer is his tittle, please add

**Comment by hpdang** [comment]
@prankshaw Software Engineer is the title of Sudhir Verma

**Comment by nityanandagohain** [comment]
chetankhatri

use this I have changed size to 300 * 300

**Comment by prankshaw** [comment]
preview -
https://prankshaw.github.io/2018-fossasia/

**Comment by hpdang** [comment]
@prankshaw please fix the conflict

**Comment by mariobehling** [comment]
@prankshaw Thanks. Please resolve merge conflicts.

**Pull request merged**